RubeesEat.IntegrationTests/PageObjects/EditBillPageObject.cs

```diff
@@ -0,0 +1,61 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using RubeesEat.IntegrationTests.WebTesting;
+
+namespace RubeesEat.IntegrationTests.PageObjects;
+
+public class EditBillPageObject : PageObject
+{
+    public string[] GetPersonAmountAndNames()
+    {
+        return GetPersonAmounts()
+               .Select(e => e.Text)
+               .ToArray();
+    }
+
+    public string? GetErrorMessage()
+    {
+        return FindElements("placeForErrorMessage").FirstOrDefault()?.Text;
+    }
+
+    public void SetNewDesciptionText(string value)
+    {
+        var element = FindElement("billDescription");
+        element.Clear();
+        element.SendKeys(value);
+    }
+
+    public void SetNewTotalPriceText(string value)
+    {
+        var element = FindElement("billAmount");
+        element.Clear();
+        element.SendKeys(value);
+    }
+    public PersonAmountControlObject ClickAddPerson()
+    { 
+        InvokeAction("addPerson");
+        return new PersonAmountControlObject(this, 
+            FindElement("addedPeople").FindElement(By.CssSelector(".personDiv:last-child")));
+    }
+
+    public void ClickUpdateBill()
+    {
+        InvokeAction("updateBill");
+    }
+
+    public PersonAmountControlObject[] GetPersonAmounts()
+    {
+        return FindElement("addedPeople").FindElements(By.CssSelector("div > .text"))
+                                         .Select(e => new PersonAmountControlObject(this, e))
+                                         .ToArray();
+    }
+
+    public List<string> GetSelection()
+    {
+        var selectElement = FindElement("billPeople");
+        var options = selectElement.FindElements(By.TagName("option"));
+        return options.Select(e => e.Text).ToList();
+    }
+    
+    
+}
```

Comment:
This is very similar to SplitBillPageObject. I think the only difference is which button can be pushed to submit.
We should create a more general PageObject Class, of which SplitBillPageObject and EditBillPageObject derive.
Or we can simply rename both buttons into submitBill. That way we only need one PageObject class which works for both